### DISCUSSION TITLE
allow user to only upload to prefix?

### DISCUSSION THREAD

**Discussion by astronautdolphindetective** [minio/minio#19505] [Q&A]
I want to create a bucket with different prefixes. Each prefix refers to one user. I want to create a policy so that the users can only upload files to their specific prefix and not into the other prefixes. Is this possible or would each user require his or her own bucket?

**Comment by klauspost** [comment]
Yes.  https://docs.aws.amazon.com/AmazonS3/latest/userguide/example-policies-s3.html#iam-policy-ex1

**Reply by astronautdolphindetective** [reply]
thank you very much for the reply. I tried that. I am using svacct add to restrict the user capabilities to certain prefixes however in this case it fails. Here is the discussion about this phenomena: https://github.com/minio/minio/discussions/19271
